outputs.tf

```diff
@@ -2,3 +2,18 @@ output "static_ip" {
   description = "The static IP address attached to the VM instance."
   value       = local.static_ip
 }
+
+output "self_link" {
+  description = "The URI of the created resource."
+  value       = google_compute_instance.vm_instance.self_link
+}
+
+output "id" {
+  description = "an identifier for the resource with format projects/{{project}}/zones/{{zone}}/instances/{{name}}"
```

Comment:
NitPick: Capital 'A' to start the description :)

Reply:
Oops!